sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/connections/one_lake_artifacts.py

```diff
@@ -23,3 +28,35 @@ class OneLakeConnectionArtifact:
     def __init__(self, *, name: str, **kwargs: Any):  # pylint: disable=unused-argument
         self.name = name
         self.type = "lake_house"
+
+
+    @classmethod
+    def _load(
+        cls,
+        data: Optional[Dict] = None,
+        yaml_path: Optional[Union[PathLike, str]] = None,
+        params_override: Optional[list] = None,
+        **kwargs: Any,
+    ) -> "Connection":
+        data = data or {}
+        params_override = params_override or []
+        context = {
+            BASE_PATH_CONTEXT_KEY: Path(yaml_path).parent if yaml_path else Path("./"),
+            PARAMS_OVERRIDE_KEY: params_override,
+        }
+        return cls._load_from_dict(data=data, context=context, **kwargs)
+
+    @classmethod
+    def _load_from_dict(cls, data: Dict, context: Dict, **kwargs: Any) -> "OneLakeConnectionArtifact":
+        loaded_data: Connection = load_from_dict(schema_class, data, context, **kwargs)
+        return loaded_data
+
+    def _to_dict(self) -> Dict:
+        # pylint: disable=no-member
+        from azure.ai.ml._schema.connections import OneLakeArtifactSchema
+        # Not sure what this pylint complaint was about, probably due to the polymorphic
+        # tricks at play. Disabling since testing indicates no issue.
+        # pylint: disable-next=missing-kwoa
+        import pdb; pdb.set_trace()
```

Comment:
Remove lingering breakpoint.

```suggestion
```


(Also pro tip, python has a [`breakpoint()`](https://www.digitalocean.com/community/tutorials/python-breakpoint) function that does the same thing)